.github/workflows/lint.yml

```diff
@@ -0,0 +1,71 @@
+name: Lint
+
+on:
+  workflow_call:
+    inputs:
+      forceSkip:
+        default: "false"
+        type: string
+      directories:
+        required: true
+        description: "Directories to lint"
+        type: string
+      cmd:
+        required: true
+        description: "command to run"
+        type: string
+
+env:
+  DEBIAN_FRONTEND: "noninteractive"
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Task
+        uses: arduino/setup-task@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install prettier
+        run: task -p deps:install:prettier
+
+      - name: Lint
+        run: task -p lint:prettier
+
+  golangci-lint:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        directory: ${{ fromJson(inputs.directories) }}
```

Comment:
Looks really ugly when this workflow is used:
```
  directories: '["."]'
```
Maybe just comma-separated, if possible?
```
  directories: dir1
  directories: dir/dir/dir, dir2/dir2/dir2
```